NuGet.config

```diff
@@ -4,6 +4,7 @@
     <!-- only use the feeds below -->
     <clear/>
     <add key="AspNetCI" value="https://www.myget.org/F/aspnetmaster/api/v3/index.json" />
+    <add key="aspnet-contrib" value="https://www.myget.org/F/aspnet-contrib/api/v3/index.json" />
```

Comment:
Nope. OpenIddict is still an alpha thingy (ASOS will likely reach RTM for Q4, but not OpenIddict)